doc/cookbook/source/examples/evaluation/cross_validation_pipeline.rst

```diff
@@ -0,0 +1,34 @@
+============================
+Cross Validation on Pipeline
+============================
+
+In this example, we illustrate how to use cross-validation with :sgclass:`CPipeline`.
+
+For more information on cross-validation, check :doc:`./cross_validation`.
+
+-------
+Example
+-------
+We'll use as example a binary classification problem solvable by a pipeline consisted of a transformer :sgclass:`CPruneVarSubMean` and a machine :sgclass:`CLibLinear`.
```

Comment:
Slightly weight English:
We demonstrate a pipeline consisting of a transformer ..., and LibLinear for binary classification.
(maybe also link to the liblinear cookbook)